lib/server/websocket/channel/server.ex

```diff
@@ -0,0 +1,154 @@
+defmodule Helix.Server.Websocket.Channel.Server do
+  @moduledoc """
+  Channel to notify all players connected to a certain server about events
+  regarding said server
+  """
+
+  use Phoenix.Channel
+
+  alias Helix.Network.Service.Henforcer.Network, as: NetworkHenforcer
+  alias Helix.Entity.Service.API.Entity, as: EntityAPI
+  alias Helix.Software.Controller.Storage, as: StorageController
+  alias Helix.Hardware.Service.API.Component, as: ComponentAPI
+  alias Helix.Hardware.Service.API.Motherboard, as: MotherboardAPI
+  alias Helix.Software.Service.API.File, as: FileAPI
+  alias Helix.Server.Service.API.Server, as: ServerAPI
+  alias Helix.Server.Service.Henforcer.Server, as: Henforcer
+
+  intercept ["notification"]
+
+  @doc false
+  def join(topic, message, socket)
+
+  # Connecting to an external server
+  def join("server:" <> server_id, %{"gateway_id" => gateway}, socket) do
+    has_connection? = fn ->
+      NetworkHenforcer.has_ssh_connection?(gateway, server_id)
+    end
+
+    with \
+      true <- Henforcer.server_exists?(server_id) || {:error, :not_found},
+      true <- Henforcer.server_exists?(gateway) || {:error, :not_found},
+      true <- Henforcer.functioning?(server_id) || {:error, :not_assembled},
+      true <- Henforcer.functioning?(gateway) || {:error, :not_assembled},
+      owner = EntityAPI.fetch_server_owner(gateway),
+      account_id = EntityAPI.get_entity_id(socket.assigns.account),
+      owner_id = EntityAPI.get_entity_id(owner),
```

Comment:
TODO: move this question (`entity owns server?`) to Entity Henforcer. But depends on #168